messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ActiveMQServerService.java

```diff
@@ -161,16 +169,12 @@ public synchronized void start(final StartContext context) throws StartException
                     ROOT_LOGGER.aioInfo();
                 }
                 configuration.setJournalType(JournalType.NIO);
+            } else if (! AIOSequentialFileFactory.isSupported(configuration.getJournalDirectory())) {
```

Comment:
A single check on AIOSequentialFileFactory.isSupported(String) is enough as it encapsulates a call to AIOSequentialFileFactory.isSupported().